app/controllers/general.php

```diff
@@ -632,6 +724,11 @@
 
 include_once __DIR__ . '/shared/api.php';
 
+App::wildcard()
```

Comment:
Wouldn't that happen anyway?

Reply:
We need it so there is `Route` for every request. Without that, ::init() wouldnt happen. Init is needed for Appwrite Proxy to kick in.